pkg/network/config/tracer_config.go

```diff
@@ -81,10 +83,24 @@ func TracerConfigFromConfig(cfg *config.AgentConfig) *Config {
 	tracerConfig.EnableMonotonicCount = cfg.Windows.EnableMonotonicCount
 	tracerConfig.DriverBufferSize = cfg.Windows.DriverBufferSize
 
+	tracerConfig.UDPConnTimeout = getUDPConnTimeout()
```

Comment:
Does this mean that the `30 sec` value set in https://github.com/DataDog/datadog-agent/blob/master/pkg/network/config/config.go#L119 will always get overwritten?

Reply:
Correct, I'm not sure what, if any value to put there. We don't read from procfs anywhere in that function, and I was trying to keep to that pattern.